File: src/app/model/question.ts
export class Question {
  id: number = 0;
  shortName: string = "";
  questionDetail: string = "";
  optionA: string = "";
  optionB: string = "";
  optionC: string = "";
  optionD: string = "";
  correctAnswer: string = "";
  reportQuestion: string = "";
  creationDate!: Date;
}

File: src/main/java/com/example/demo/Entity/Question.java
package com.example.demo.Entity;

import java.util.Date;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.ManyToOne;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Question
{
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	private String shortName;
	private String questionDetail;
	private String optionA;
	private String optionB;
	private String optionC;
	private String optionD;
	private String correctAnswer;
	private String reportQuestion;
	private Date creationDate;
	private int isDeleteQuestion; //0=active 1=delete
	
	
	@ManyToOne
	User whoAddedQuestion;
}
